tests/e2e/scripts/rancher.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+node_ip=$1
+
+curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
```

Comment:
Maybe just drop a manifest?
```yaml
---
apiVersion: v1
kind: Namespace
metadata:
  name: cert-manager
---
apiVersion: v1
kind: Namespace
metadata:
  name: cattle-system
---
apiVersion: helm.cattle.io/v1
kind: HelmChart
metadata:
  namespace: kube-system
  name: cert-manager
spec:
  targetNamespace: cert-manager
  version: v1.6.1
  chart: cert-manager
  repo: https://charts.jetstack.io
  set:
    installCRDs: "true"
---
apiVersion: helm.cattle.io/v1
kind: HelmChart
metadata:
  name: rancher
  namespace: kube-system
spec:
  targetNamespace: cattle-system
  version: 2.6.5
  chart: rancher
  repo: https://releases.rancher.com/server-charts/latest
  set:
    ingress.tls.source: "rancher"
    hostname: "localhost"
    antiAffinity: "required"
    replicas: 1
```

Reply:
This is a much better idea, I have ripped out the helm install and just embedded a manifest into the rancher.sh script.